projects/manager/src/app/datasets/add-dataset/add-dataset.component.html

```diff
@@ -0,0 +1,82 @@
+<h2 mat-dialog-title>Create new dataset</h2>
+
+<div class="container">
+    <form [formGroup]="form">
+        <mat-card>
+            <mat-card-header>
+                <mat-card-title>Basic Information</mat-card-title>
+            </mat-card-header>
+
+            <mat-card-content>
+                <div>
+                    <mat-form-field appearance="fill">
+                        <mat-label>Name</mat-label>
+                        <input matInput type="text" formControlName="name" />
+                    </mat-form-field>
+                    <mat-form-field appearance="fill">
+                        <mat-label>Display Name</mat-label>
+                        <input matInput type="text" formControlName="displayName" />
+                    </mat-form-field>
+                </div>
+                <div>
+                    Source Operator:
+                    <mat-button-toggle-group
+                        id="source-type"
+                        [multiple]="false"
+                        formControlName="sourceOperator"
+                        (ngModelChange)="updateSourceOperator()"
+                    >
+                        <mat-button-toggle [value]="SourceOperators.GdalSource">Gdal (Rasters)</mat-button-toggle>
+                        <mat-button-toggle [value]="SourceOperators.OgrSource">Ogr (Vectors)</mat-button-toggle>
+                    </mat-button-toggle-group>
+                </div>
+            </mat-card-content>
+        </mat-card>
+
+        <mat-card>
+            <mat-card-header>
+                <mat-card-title>Data Path</mat-card-title>
+            </mat-card-header>
+
+            <mat-card-content>
+                <div>
+                    Type of Data Path:
+                    <mat-button-toggle-group
+                        id="path-type"
+                        [multiple]="false"
+                        formControlName="dataPathType"
+                        (ngModelChange)="updateDataPath()"
+                    >
+                        <mat-button-toggle [value]="DataPaths.Upload">Upload</mat-button-toggle>
+                        <mat-button-toggle [value]="DataPaths.Volume">Volume</mat-button-toggle>
+                    </mat-button-toggle-group>
+
+                    <mat-form-field appearance="fill">
+                        @if (form.controls.dataPathType.value === DataPaths.Upload) {
+                            <mat-label>Upload Id</mat-label>
+                        } @else {
+                            <mat-label>Volume Name</mat-label>
+                        }
+                        <input matInput type="text" formControlName="dataPath" />
+                    </mat-form-field>
+                </div>
+            </mat-card-content>
+        </mat-card>
+
+        <mat-card>
+            <mat-card-header>
+                <mat-card-title>Loading Info</mat-card-title>
```

Comment:
I can add a vector loading info and have raster selected (was by accident).
It creates the dataset, but when I load it, it says
```json
{"error":"Operator","message":"Operator: LoadingInfoError: Could not load meta data: Invalid type: expected OgrMetaData found GdalStatic"}
```

Reply:
that's bad. We don't check the loading info in the backend currently, and I don't think it's easy to do.

I would leave it like it is and when we add concrete components for the inidividual loading info types it can no longer happen (at least in the frontend)